charts/values.yaml

```diff
@@ -59,6 +57,7 @@ fullnameOverride: ""
 
 # Configure Marklogic Admin Username and Password
 auth:
+  secretName: ""
   adminUsername: "admin"
   adminPassword: ""
   walletPassword: ""
```

Comment:
I guess adminPassword and walletPassword entries can be removed from the values.yaml file as they are no more used.
We should put a enabled statement for the walletPassword because currently the walletPassword will be set automatically even if is not needed.
We should put a comment to mention that a secret will be automatically generated if secretName is not provided.
We should put a comment to mention the format of the secret expected if we provide secretName.